Pylint: Fix `superfluous-parens` issues.

## 💡 Description
Update the codebase to remove superfluous parens in logical expressions. `pylint` report is attached (`pylint . --disable=all --enable=superfluous-parens`):
```
************* Module pds.naif_pds4_bundler.classes.list
pds/naif_pds4_bundler/classes/list.py:150:0: C0325: Unnecessary parens after 'if' keyword (superfluous-parens)
pds/naif_pds4_bundler/classes/list.py:270:0: C0325: Unnecessary parens after 'if' keyword (superfluous-parens)
************* Module pds.naif_pds4_bundler.classes.product.product_metakernel
pds/naif_pds4_bundler/classes/product/product_metakernel.py:367:0: C0325: Unnecessary parens after 'if' keyword (superfluous-parens)
```

--- a/src/pds/naif_pds4_bundler/classes/list.py
+++ b/src/pds/naif_pds4_bundler/classes/list.py
@@ -147,7 +147,7 @@ class KernelList:
                         #
                         if patterns:
                             for el in patterns:
-                                if ("$" + el) in description or ("$" + el) in mapping:
+                                if "$" + el in description or "$" + el in mapping:
                                     value = patterns[el]
 
                                     #
@@ -267,7 +267,7 @@ class KernelList:
 
                         if options:
                             for option in options.split():
-                                if ("$" + "PHASES") in option:
+                                if "$" + "PHASES" in option:
                                     if hasattr(self.setup, "phases"):
                                         if list(self.setup.phases.keys())[0]:
                                             phases = self.setup.phases["phase"]["@name"]

--- a/src/pds/naif_pds4_bundler/classes/list.py
+++ b/src/pds/naif_pds4_bundler/classes/list.py
@@ -147,7 +147,7 @@ class KernelList:
                         #
                         if patterns:
                             for el in patterns:
-                                if "$" + el in description or "$" + el in mapping:
+                                if f'${el}' in description or f'${el}' in mapping:
                                     value = patterns[el]
 
                                     #
@@ -267,7 +267,7 @@ class KernelList:
 
                         if options:
                             for option in options.split():
-                                if "$" + "PHASES" in option:
+                                if '$PHASES' in option:
                                     if hasattr(self.setup, "phases"):
                                         if list(self.setup.phases.keys())[0]:
                                             phases = self.setup.phases["phase"]["@name"]

--- a/src/pds/naif_pds4_bundler/classes/product/product_metakernel.py
+++ b/src/pds/naif_pds4_bundler/classes/product/product_metakernel.py
@@ -364,7 +364,7 @@ class MetaKernelProduct(Product):
                 #
                 if patterns:
                     for el in patterns:
-                        if ("$" + el) in description:
+                        if "$" + el in description:
                             value = patterns[el]
 
                             #

--- a/src/pds/naif_pds4_bundler/classes/product/product_metakernel.py
+++ b/src/pds/naif_pds4_bundler/classes/product/product_metakernel.py
@@ -364,7 +364,7 @@ class MetaKernelProduct(Product):
                 #
                 if patterns:
                     for el in patterns:
-                        if "$" + el in description:
+                        if f'${el}' in description:
                             value = patterns[el]
 
                             #

